scipy/stats/mstats_extras.py

```diff
@@ -392,7 +392,50 @@ def compare_medians_ms(group_1, group_2, axis=None):
         ndarray of floats with a length equal to the length of `group_1`
         along `axis`.
 
+    Examples
+    --------
+    >>> from scipy import stats
+    >>> a = [1, 2, 3, 4, 5, 6, 7]
+    >>> b = [8, 9, 10, 11, 12, 13, 14]
+
+    Arrays must be at least of size 7.
+
+    >>> stats.mstats.compare_medians_ms(a, b, axis=None)
+    1.0693225866553746e-05
+
+    Since no axis is provided, the result is flattened.
+
+    The McKean–Schrader estimate of the standard error is used 
+    in conjunction with a Welch-type test in hypothesis testing. 
+    It is used for the special case of testing the hypothesis of equal medians, 
+    when Yuen-Welch and Box methods are not recommended.
+    [1]_ Wilcox.
+
+    >>> c = [-5, 82, 11, 11, 22, 55, 90]
+    >>> stats.mstats.compare_medians_ms(b, c)
+    0.275814234966238
+
+    The function is vectorized to compute along a given axis.
+    Numpy Broadcasting compares the shapes of two arrays element-wise. 
+    Two dimensions are compatible when they are equal or one of them is 1.
```

Comment:
```suggestion
```
We have two options - rely on the user to be familiar with NumPy broadcasting rules, which are used throughout SciPy, or include statements like this in every function that has an `axis` argument. In other functions with `axis`, you won't find this sort of explanation, so let's remove it here. If you think it would be helpful in every function with an `axis` argument, that is something that could be considered - but it would be a separate PR.